packages/ses/src/commons.js

```diff
@@ -266,3 +266,9 @@ export const FERAL_EVAL = eval;
 // Sample at module initialization time, which is before lockdown can
 // repair it.  Use it only to build powerless abstractions.
 export const FERAL_FUNCTION = Function;
+
+export const noEvalEvaluate = () => {
+  throw new TypeError(
+    'Cannot eval with evalTaming set to "noEval" (SES_NO_EVAL)',
```

Comment:
We should add a `packages/ses/error-codes/SES_NO_EVAL.md` with information for people who encounter this error and how to mitigate it.